src/main/java/net/rptools/maptool/client/ui/campaignproperties/CampaignPropertiesDialog.java

```diff
@@ -529,4 +553,354 @@ public static void main(String[] args) {
+    }
+
+    /* Parameterised strings - need to be done individually */
+    parameters.put(
+        "wikiLinkReferral",
+        I18N.getText(
+            "sightLight.wikiLinkReferral",
+            "<i>wiki.rptools.info/index.php/Introduction_to_Lights_and_Sights</i>"));
+    if (MapTool.getLanguage().toLowerCase().startsWith("en")) {
+      /* remove translated version of words for English locales. */
+      parameters.put(
+          "optionDescriptionShape",
+          I18N.getText("sightLight.optionDescription.shape", "", "", "", "", "", "")
+              .replace("(", "")
+              .replace(")", ""));
+    } else {
+      parameters.put(
+          "optionDescriptionShape",
+          I18N.getText(
+              "sightLight.optionDescription.shape",
+              parameters.get("shapeTypeNameBeam"),
+              parameters.get("shapeTypeNameCircle"),
+              parameters.get("shapeTypeNameCone"),
+              parameters.get("shapeTypeNameGrid"),
+              parameters.get("shapeTypeNameHexagon"),
+              parameters.get("shapeTypeNameSquare")));
+    }
+    parameters.put(
+        "optionDescriptionPersonalSightComponentColor",
+        I18N.getText("sightLight.optionDescription.personalSight.component.color", "#rrggbb"));
+    parameters.put(
+        "optionDescriptionRestriction",
+        I18N.getText("sightLight.optionDescription.restriction", "gm", "owner"));
+
+    /* everything else */
+    helpKeys = I18N.getMatchingKeys("sightLight");
+    for (String key : helpKeys) {
+      parameters.putIfAbsent(
+          uncapitalize(
+              capitalize(key, new char[] {'.'}).replace("SightLight", "").replace(".", "")),
+          I18N.getText(key));
+    }
+    return parameters;
+  }
+
+  /**
+   * Creates HTML for both sight and light help
+   *
+   * @return String[]
+   */
+  private String[] generateHelpText() {
+    Map<String, String> parameters = createSightLightHelpTextMap();
+    String structureCommon =
+        """
+            <html>
+            <body >
+            <font size=4>${wikiLinkReferral}</font><br>
+            <u><font size=5>${subheadingStructure}</font></u><br><br>
+            <ul compact>
+            <li>${structureListItemLines}</li>
+            <li>${structureListItemMeasurement}</li>
+            <li>${structureListItemDefaults}</li>
+            <li>${structureListItemComments}</li>
+            <li>${structureListItemLetterCase}</li>
+            """;
+    String structureLight =
+        """
+            <li>${structureListItemMultiple}<sup>1</sup></li>
+            <li>${structureListItemGroupName}</li>
+            <li>${structureListItemGroupedNames}</li>
+            <li>${structureListItemGroups}</li>
+            <li>${structureListItemSorting}</li>
+            """;
+    String syntaxHeading =
+        """
+              </ul>
+              <u><font size=5>${subheadingDefinitionSyntax}</font></u><br><br>
```

Comment:
I generally dislike the idea of having the closing syntax (here < / u l > in 658) in a separate string from the beginning (here < u l > in 641). This is side-effect programming and hard to follow and latter to maintain valid html, when names and other items change. Line 640 is better.